### PR TITLE
feat: allow empty console

### PR DESCRIPTION
**Pull request by moodmosaic**
*Continuation of #367, reworked (as per @obycode in https://github.com/hirosystems/clarinet/pull/367#issuecomment-1135035971) in order to take #333 into account.*

I assume that if the manifest path is empty, warn the user, skip deployments and artifacts, and just start an empty console session.

Output from `clarinet console`:
image

### CODE REVIEW

**Comment by obycode** [comment]
Thanks for the updates! I'll take a look this evening.

> Hmmm, I'm pretty sure `cargo fmt --all -- --check` completed without warnings on my local. 🤔 Unsure why CI failed.

Looks like the error this time is from `cargo audit`, unrelated to your changes. We'll ignore that for this PR and fix it separately.

File: src/frontend/cli.rs [hunk 1]
```diff
@@ -992,6 +997,18 @@ fn get_manifest_path_or_exit(path: Option<String>) -> PathBuf {
     }
 }
 
+fn get_manifest_path_or_warn(path: Option<String>) -> Option<PathBuf> {
+    match get_manifest_path(path) {
+        Some(manifest_path) => Some(manifest_path),
+        None => {
+            println!(
+              "{}: no --manifest-path specified, so a plain REPL will be started with no contracts loaded.",
```
[the remark this answers is not in the document]
Comment (moodmosaic) [reply src/frontend/cli.rs:1005]: Sure! If you think that's better. (I don't know precisely what 'default' manifest means.) Happy to update the warning text though.

[the remark this answers is not in the document]
Comment (obycode) [reply src/frontend/cli.rs:1005]: Yeah, I didn't love "default" when I wrote that either, but couldn't think of a better way to word it ("Continuing with an empty session"?). I think it should get the point across, unless we have a better way of saying it simply. The important part is that the manifest was not found, so if you see this message and expected to find a manifest, you'll know something went wrong.

**Review by obycode — APPROVED** [review]
Looks great, thanks @moodmosaic!

**Pull request merged**